src/components/core/geoChart/tests/geoChartDataLayers.spec.ts

```diff
@@ -38,6 +39,84 @@ describe("createPushpinDataLayer", () => {
         ]);
     });
 
+    it("should return default border, color and size when Color, Size contains same values", () => {
+        const geoData: IGeoData = {
+            size: {
+                index: 0,
+                name: "size",
+            },
+            color: {
+                index: 1,
+                name: "color",
+            },
+            location: {
+                index: 0,
+                name: "location",
+            },
+        };
+        const excutionResult: Execution.IExecutionResult = {
+            data: [Array(3).fill(1), Array(3).fill(10)],
+            headerItems: [
+                [
+                    [
+                        {
+                            measureHeaderItem: {
+                                name: "Size",
+                                order: 0,
+                            },
+                        },
+                        {
+                            measureHeaderItem: {
+                                name: "Color",
+                                order: 1,
+                            },
+                        },
+                    ],
+                ],
+                [
+                    [
+                        {
+                            attributeHeaderItem: {
+                                name: "44.500000;-89.500000",
+                                uri: "/gdc/md/projectId/obj/694/elements?id=1808",
+                            },
+                        },
+                        {
+                            attributeHeaderItem: {
+                                name: "44.500000;-89.500000",
+                                uri: "/gdc/md/projectId/obj/694/elements?id=1809",
+                            },
+                        },
+                        {
+                            attributeHeaderItem: {
+                                name: "44.500000;-89.500000",
+                                uri: "/gdc/md/projectId/obj/694/elements?id=1810",
+                            },
+                        },
+                    ],
+                ],
+            ],
+            paging: {
+                count: [2, 50],
+                offset: [0, 0],
+                total: [2, 50],
```

Comment:
this should equal to [line](https://github.com/gooddata/gooddata-react-components/pull/1399/files#diff-08a085e86141af62510a25c0525d0ffeR58) 
```
count: [2, 10],
offset: [0, 0],
total: [2, 10],
```

Reply:
fixed